### PR TITLE
feat(daikon-tql-client): manage tql options

### PR DESCRIPTION
**Pull request by ncomont**
- between operator (excludeMin, excludeMax)
- parser (options getters)

**What is the problem this Pull Request is trying to solve?**
No way to exclude max or min values in between operator.
Parser can't get operator options. 

**What is the chosen solution to this problem?**
New `excludeMin` and `excludeMax` options for `between` operator.
Parser refacto.

**Please check if the Pull Request fulfills these requirements**
- [x] The PR commit message follows our [guidelines](../CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features, coverage should be over 75% in the new code)
- [x] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in JIRA), if any, are all linked or available in the Pull Request

 
**[ ] This Pull Request introduces a breaking change**

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/314/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/314/)

File: daikon-tql/daikon-tql-client/src/converter/parser.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ const mapping = {
 	inside_range: {
 		operator: 'between',
 		getValues: node => node.args.intervals,
-		getOptions: () => {},
+		getOptions: ({ excludeMax, excludeMin }) => ({ excludeMax, excludeMin }),
```
Comment (frassinier) [inline daikon-tql/daikon-tql-client/src/converter/parser.js]: I don't understand?
You get an object and you return an object?

Comment (ncomont) [reply daikon-tql/daikon-tql-client/src/converter/parser.js]: I get the whole node object, so I extract the only values I'm interested by : 


Same function in ES5 could be :

```javascript
function ish({ excludeMax, excludeMin }) {
	return {
		excludeMax: item.excludeMax,
		excludeMin: item.excludeMin,
	};
}
```


So :

```javascript
var sblah = {
	excludeMax: true,
	excludeMin: false,
	yo: 'lo',
	pumpum: 'ratatata',
};

ish(sblah);
```

Will give :

```javascript
{
	excludeMax: true,
	excludeMin: false,
}
```

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/315/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/315/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/4/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/4/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/8/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/8/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/10/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/10/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/11/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/11/)

**Pull request merged**